rust/src/detect/xbits.rs

```diff
@@ -0,0 +1,224 @@
+use std::ffi::{CStr, CString};
+use std::os::raw::{c_char, c_void};
+
+pub const DETECT_XBITS_CMD_SET: u8 = 0;
+pub const DETECT_XBITS_CMD_TOGGLE: u8 = 1;
+pub const DETECT_XBITS_CMD_UNSET: u8 = 2;
+pub const DETECT_XBITS_CMD_ISNOTSET: u8 = 3;
+pub const DETECT_XBITS_CMD_ISSET: u8 = 4;
+pub const DETECT_XBITS_CMD_NOALERT: u8 = 5;
+pub const DETECT_XBITS_CMD_MAX: u8 = 6;
+pub const DETECT_XBITS_TRACK_IPSRC: u8 = 0;
+pub const DETECT_XBITS_TRACK_IPDST: u8 = 1;
+pub const DETECT_XBITS_TRACK_IPPAIR: u8 = 2;
+pub const DETECT_XBITS_TRACK_FLOW: u8 = 3;
+pub const DETECT_XBITS_EXPIRE_DEFAULT: u32 = 30;
+
+#[repr(C)]
+#[derive(Debug)]
+pub struct RSXBitsData {
```

Comment:
I think we should drop the `RS` prefix unless its to avoid some naming conflict.